portfolio/src/main/webapp/blogStyle.css

```diff
@@ -0,0 +1,245 @@
+html {
+    scroll-behavior: smooth
+  }
+  
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style: none;
+    text-decoration: none;
+    font-family: 'Raleway', sans-serif;
+  }
+  
+  body {
+     background-color: #f3f5f9;
+     overflow-x: hidden;
+  }
+  
+  /* -------------- Toggle VIew of Side Bar --------------*/
+  
+  input[type="checkbox"] {
+    position: fixed;
+    right: 20px;
+    top: 20px;
+    width: 50px;
+    height: 50px;
+    z-index: 100000;
+    cursor: pointer;  
+    opacity: 0;
+  }
+  
+  .icon {
+    position: fixed;
+    right: 20px;
+    top: 20px;
+    width: 50px;
+    height: 50px;
+    z-index: 100001;
+    background: #6767b5;
+    pointer-events: none;
+    opacity: 80%;
+  }
+  
+  .icon i:before {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom:0;
+    color: #fff;
+    text-align: center;
+    line-height: 50px;  
+  }
+  
+  /* -------------- Side Bar --------------*/
+  
+  .sidebar {
+    position: fixed;
+    top: 0;
+    left: 0;
+    margin: 0;
+    padding: 30px 0px;
+    box-sizing: border-box;
+    width: 300px;
+    height: 100%;
+    background: #6767b5;
+    transition: 0.5s;
+    z-index: 1000000;
+  }
+  
+  input[type="checkbox"]:checked ~ .sidebar {
+    left: -300px;
+  }
+  
+  /* -------------- Side Bar Avatar --------------*/
+  
+  .sidebar img {
+    display: block;
+    overflow: hidden;
+    margin: 50px auto 30px auto;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;    
+    border-radius: 50%;
+  }
+  
+  /* -------------- Side Bar Headings --------------*/
+  
+  .sidebar h2 {
+    color: #fff;
+    text-align: center;
+    margin-bottom: 30px;
+  }
+  
+  .sidebar h4 {
+    color: #fff;
+    text-align: center;
+    margin-bottom: 30px;
+  }
+  
+  /* -------------- Side Bar Menu --------------*/
+  
+  .tab {
+    border-top: 1px solid rgba(255,255,255,0.05);
+    border-bottom: 1px solid rgba(0,0,0,0.05);
+  }
+  
+  .tab a {
+    padding: 15px;
+    color: #bdb8d7;
+    display: block;
+    padding-left: 30px;
+  }
+  
+  .tab a .fas {
+    width: 25px;
+  }
+  .tab:hover {
+    background-color: #40317d;
+  }
+  
+  .tab:hover a {
+    color: #fff;
+  }
+  
+  .tab a.active{
+    background: #00b5de;
+    color: white;
+  }
+  
+  /* -------------- Social Media --------------*/
+  
+  .social_media ul {
+    list-style: none;
+    margin: 0 10px;
+    position: relative;
+    margin-bottom: 25px;
+    left: 15%;
+    top: 50%;
+    transform: translateX(-50%, -50%);
+    display: flex;
+  }
+  
+  .social_media li {
+      margin: 0 2px;
+  }
+  
+  .social_media a {
+    display: block;
+    width: 30px;
+    background: #fff;
+    height: 30px;
+    line-height: 35px;
+    text-align: center;
+    margin: 0 5px;
+    color: #bdb8d7;
+    border-radius: 50%;
+    transition: 0.5s;
+    transform: translate(0,0px);
```

Comment:
This rule seems a bit suspicious - you want to move it 0px? The side-effect of doing the transform is that it creates a new stacking context, so it's not entirely a no-op, but that implies something weird is going on if this is necessary.